### ISSUE TITLE
slides

### ISSUE THREAD

**Issue by michaellindon** [Chasinggoal/Data-_learning_search#2]
I have some feedback for the slides:

Feature extractions:
Size height width type are relevant features, but they are not the most interesting and my intuition not the most relevant. I would rather the audience know about things like keyword in url, tf-idf frequency of keyword in page content, keyword in image name, keyword in html image tags. We had a long but non-exhaustive list of which features we were inputting to the model. Height and width are not really representative of how many features google uses to determine rankings. Please include all the terms we discussed. I also don't want to include moz pagerank and moz domain rank, as there is no way that google is using these features in its ranking whatsoever. You can introduce this section by talking about the large industry of SEO which knows what features to change in order to get higher rankings.

Model:
Please have a slide with the PL likelihood, and explain that we are doing gradient descent

**Comment by Chasinggoal** [comment]
Image Info: size, type, alt (contain keyword), width, height
Page Info: domain url(keyword in?), page url(keyword in?), load speed, Number of Inbound Links, HeaderTags(keyword in?), (MOZ?);

**Comment by Chasinggoal** [comment]
One question: if not using MOZ, how are we going to access the page rank thing?

**Comment by michaellindon** [comment]
two possible answers to this, we use historic page rank values, i.e. the last update value of pagerank that that domain has, or,

page rank is a function of the linking profile to a domain, when all the raw linking information is included in the model, this may explain the variability in the data and page rank may be a redundant feature

**Comment by michaellindon** [comment]
I think we had a list on the board of features, i took a photo, refer to this when making slides

**Comment by Chasinggoal** [comment]
Yes, I also have the photo, and I will put more features on PPT.
By the way, do you think this is helpful?
<img width="819" alt="2016-06-09 10 48 25" src="https://cloud.githubusercontent.com/assets/16568976/15933986/df131684-2e2f-11e6-8012-9c6a26b53e74.png">

**Comment by michaellindon** [comment]
it looks really cool, but im not sure exactly what that is, is it whether a domain has a facebook or twitter account, and how many tweets it has? or is it how many tweets have linked to that domain? I'd like to make use of it, but need a clearer definition of what it is

**Comment by Chasinggoal** [comment]
I am not 100% sure. I am not able to access this up to now. I need to use it to check out the JSON info returned from MOZ to see what exactly it is. But I will write an email to inquire for that.
